db.go

```diff
@@ -86,6 +92,10 @@ func (d *db) Insert(structPtr interface{}) (sql.Result, error) {
 	return d.s.Insert(structPtr)
 }
 
+func (d *db) InsertContext(ctx context.Context, structPtr interface{}) (sql.Result, error) {
```

Comment:
Please rename added functions to `XXXWithContext`. That's go way.

Reply:
Fixed in 175a576